Source/Core/Common/FileUtil.cpp

```diff
@@ -745,6 +745,23 @@ std::string& GetExeDirectory()
 	}
 	return DolphinPath;
 }
+#else
+std::string& GetExeDirectory()
```

Comment:
Half of the code here is shared with the windows implementation. It should be fine to only rewrite the inner part within if{}.